lib/workers/repository/error-config.ts

```diff
@@ -49,6 +49,7 @@ export async function raiseConfigWarningIssue(
       body,
       once,
       shouldReOpen: shouldReopen,
+      confidential: config.confidential,
```

Comment:
I'm not sure if a config warning issue should be confidential?

Reply:
Hi @rarkins I guess keeping this issue confidential is good for project security - malicious users could use this config issues info for some purposes.